Bank/dbo/Tables/Clients.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE Clients
+(
+	Id INT IDENTITY(1,1) NOT NULL,
+	FirstName VARCHAR(25) NOT NULL,
+	SecondName VARCHAR(25) NOT NULL,
+	Age INT NOT NULL,
+	Phone VARCHAR(25) NOT NULL,
+	Email VARCHAR(25),
+	Bank_Id INT NOT NULL,
+	Social_Status_Id INT NOT NULL,
+	PRIMARY KEY CLUSTERED ([Id] ASC),
+	CONSTRAINT FK_Clients_To_Bank FOREIGN KEY(Bank_Id) REFERENCES Banks(Id) ON DELETE CASCADE,

```

Comment:
If you will have the same person in the other bank, you will create another client
It should be that one client can have different banks, and one bank can have different clients